js-foundations.js

```diff
@@ -3,28 +3,60 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
+function numChecker(num) {
+    return (num % 3 === 0 ? `${num} is divisible by three.` : `${num} is not divisible by three.`)
+}
+
+console.log(numChecker(15))
+console.log(numChecker(0))
+console.log(numChecker(385))
+console.log(numChecker(-7))
 
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
+var helloMe = {
+    firstName: "Yumi",
+    lastName: "Park",
+    ethnicity: "Korean-American",
+    hometown: "Los Angeles",
+    getData: function() { return "My name is " + this.firstName + " " + this.lastName + ". I am " + this.ethnicity + " and I am from " + this.hometown + "." }
+}
 
+console.log(helloMe.getData())
 
```

Comment:
Great job! You could use string interpolation here to make your code a little cleaner. Just a suggestion, totally optional.